src/client/components/pages/entities/work.js

```diff
@@ -90,7 +133,10 @@ function WorkDisplayPage({entity, identifierTypes, user}) {
 				</Col>
 				<Col lg={10}>
 					<EntityTitle entity={entity}/>
```

Comment:
This EntityTitle is missing the prop `handleModalToggle={handleModalToggle}`

Reply:
Added